examples/bri-3/src/bri/workgroup/workgroups/api/workgroups.controller.spec.ts

```diff
@@ -23,16 +23,16 @@ import { classes } from '@automapper/classes';
 
 describe('WorkgroupsController', () => {
   let workgroupController: WorkgroupController;
-
   let mockBpiSubjectStorageAgent: MockBpiSubjectStorageAgent;
   let mockWorkgroupStorageAgent: MockWorkgroupStorageAgent;
 
+  const tokenHeader =
```

Comment:
@Kasshern presume the hard coded values are only there for testing and until the DB runs. Two things:

- please, either mark as a ToDo wherever there is a hard-coded value to be moved into either a separate test setup or a configuration file
- or move the hardcoded values into a configuration file that is loaded into the module

Reply:
Although this file has been altered these hardcoded values are used to instantiate entities within mock storage. Each controller spec tests uses hardcoded entities for mocks of storage.

Reply:
Issue created